src/main/java/com/code_intelligence/jazzer/Jazzer.java

```diff
@@ -265,7 +266,42 @@ private static Path javaBinary() {
     return Paths.get(System.getProperty("java.home"), "bin", javaBinaryName);
   }
 
-  private static Stream<String> javaBinaryArgs() {
+  private static Stream<String> javaBinaryArgs() throws IOException {
+    if (IS_ANDROID) {
+      // Add Android specific args
+      Path agentPath =
+          RulesJni.extractLibrary("android_native_agent", "/com/code_intelligence/jazzer/android");
+
+      String jazzerAgentPath = System.getProperty("jazzer.agent_path");
```

Comment:
Just FYI: I will clean up these usages in a follow-up PR as part of an ongoing Opt cleanup.